contracts/priv/cloud-priv.yml

```
@@ -619,6 +736,66 @@ components:
                     user:
                       readOnly: true
                       $ref: '#/components/schemas/Link'
+    OnboardingWithIDsRequest:
+      type: object
+      properties:
+        user:
+          $ref: '#/components/schemas/Identity'
+        org:
+          $ref: '#/components/schemas/Identity'
+        config:
+          type: object
+          properties:
+            bucket:
+              type: string
+              description: name of non system bucket
+            retentionPeriodSeconds:
+              type: integer
+              description: retention period of non system bucket
+            limit:
+              $ref: '#/components/schemas/Limit'
+        returnToken:
+          type: bool
+          description: if true will return the auth token in the response
+      required:
+        - username
+        - org
+        - config
+    OnboardingUserRequest:
+      type: object
+      properties:
+        user:
+          $ref: '#/components/schemas/Identity'
+        orgID:
+          type: string
+          description: organization id to add user to
+        role:
+          type: string
+          description: Role of the User
+          enum:
+            - owner
+            - member
+      required:
+        - user
+        - orgID
+        - Role
+    OnboardingWithIDsResponse:
+      type: object
+      properties:
+        user:
+          $ref: '#/components/schemas/Identity'
+        org:
+          $ref: '#/components/schemas/IdentityLinks'
```

```suggestion
          $ref: '#/components/schemas/Identity'
```

We don't need links for this response...

---

That is what is in the specs on the issue